lib/zendesk_api/middleware/response/sanitize_response.rb

```diff
@@ -0,0 +1,21 @@
+module ZendeskAPI
+  module Middleware
+    module Response
+      class SanitizeResponse < Faraday::Response::Middleware
+        def on_complete(env)
+          to_utf8!(env[:body])
+        end
+
+        def to_utf8!(str)
+          str.force_encoding(Encoding::UTF_8)
+          if str.respond_to?(:scrub)
+            str.scrub!('') unless str.valid_encoding?
```

Comment:
If we can just assume that that won't be a problem with some customers that might be using the gem using the old 1.9.3 version, I am more than fine with it.